requirements.txt

```diff
@@ -0,0 +1,42 @@
+# Sphinx dependencies
+Sphinx>=7.0.0
+sphinx_rtd_theme==2.0.0
+
+ase>=3.18.0
+contourpy>=1.2.0
+coverage>=7.5.0
+cycler>=0.12.0
+exceptiongroup>=1.2.0
+fonttools>=4.53.0
+importlib_resources>=6.4.0
+iniconfig>=2.0.0
+kiwisolver>=1.4.5
+matplotlib>=3.9.0
+
+metatensor==0.2.0
+metatensor-core==0.1.8
+metatensor-learn==0.2.2
+metatensor-operations==0.2.2
```

Comment:
Do we want to pin these to specific versions? It is usually good to use >= whenever we can

Reply:
I can update these, I was just worried about stability issues from possible updates.